src/bot.ts

```diff
@@ -100,22 +100,52 @@ const onMessage = async (ctx: OnMessageContext) => {
       return voiceMemo.onEvent(ctx).catch((e) => payments.refundPayment(e, ctx, price));
     }
   }
+  if (openAiBot.isSupportedEvent(ctx)) {
+    const price = openAiBot.getEstimatedPrice(ctx);
+    if (price > 0) {
+      await ctx.reply(`Processing withdraw for ${price.toFixed(2)}¢...`);
```

Comment:
Do we need to post a reply about payment after every message from client? Just asking because I'm not sure

Reply:
I did it to tell the user how much we are charging (the message it's received once)... Let's keep it there, wait for what they say, and not change the rest of the bots.